web/src/engine/attachment/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#
+
+# set -x
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
+
+# Imports common Web build-script definitions & functions
+SUBPROJECT_NAME=engine/attachment
+. "$KEYMAN_ROOT/web/common.inc.sh"
+
+# ################################ Main script ################################
+
+builder_describe "Builds the Keyman Engine for Web (KMW) attachment engine." \
+  "@/web/src/engine/dom-utils" \
+  "@/web/src/engine/element-wrappers" \
+  "clean" \
+  "configure" \
+  "build" \
+  "test"
+
+# Possible TODO?
+# "upload-symbols   Uploads build product to Sentry for error report symbolification.  Only defined for $DOC_BUILD_EMBED_WEB" \
+
+builder_describe_outputs \
+  configure   /node_modules \
+  build       /web/build/$SUBPROJECT_NAME/lib/index.mjs
+
+builder_parse "$@"
+
+#### Build action definitions ####
+
+if builder_start_action configure; then
+  verify_npm_setup
+
+  builder_finish_action success configure
+fi
```

Comment:
```suggestion
builder_run_action configure verify_npm_setup
```

Reply:
I'm glad that the shorthand's now a thing.

I did go through a round of such changes with #8830, but there's little reason I can't go ahead and port those changes here.